docker-compose.yml

```diff
@@ -1,30 +1,26 @@
 services:
   api:
-    image: mcsystems/news-search-api
-    build: .
+    image: mcsystems/news-search-api:${IMAGE_TAG}
+    environment:
+      INDEXES: ${INDEXES}
+      ESHOSTS: ${ESHOSTS}
+      ESOPTS: ${ESOPTS}
+      ELASTICSEARCH_INDEX_NAME_PREFIX: ${ELASTICSEARCH_INDEX_NAME_PREFIX}
+      TERMFIELDS: ${TERMFIELDS}
+      TERMAGGRS: ${TERMAGGRS}
     ports:
-      - 8000:8000
-    networks:
-      - story-indexer
+      - ${API_PORT}:8000
```

Comment:
can you use `${API_BASE_PORT}` instead of 8000 here?

Reply:
`uvicorn.run()` defaults to 8000, wouldn't this imply our api port is a variable, but that does reflect [here ](https://github.com/mediacloud/news-search-api/blob/3ef32bb7f49445fb188f61b2741822875bf9642f/api.py#L651). Unless we foresee a need to run the api in another port?

Reply:
You're right. Nevermind!